accounts/aws/patch-aws.yml

```diff
@@ -8,7 +8,22 @@ kind: SpinnakerService
 metadata:
   name: spinnaker
 spec:
-  spinnakerConfig:
+  spinnakerConfig:  
+    profiles:  
+    # Enable this setting under core_config/patch-deck.yml
+    #   deck:                                     # Enables Lambda Functions UI
+    #     settings-local.js: |
+    #       window.spinnakerSettings.feature.functions = true
+      clouddriver:                              # Enables Lambda Functions in "Infrastructure"
+        aws:
+          features:
+            lambda:
+              enabled: true
+          accounts:
+          - name: aws-dev                        # NOTE: profiles.clouddriver.aws.accounts is merged with config.provider.aws.accounts by INDEX not accounts.name
+            lambdaEnabled: true
+            accountId: "XXXXXXXXXXXX"            # (Required). Your AWS account ID to manage. See the AWS IAM User Guide for more information.
+            assumeRole: role/spinnaker           # (Required). If set, will configure a credentials provider that uses AWS Security Token Service to assume the specified role. Example: “user/spinnaker” or “role/spinnakerManaged”
```

Comment:
Should this `profiles` section be in `accounts/aws/patch-aws-lambda-plugin.yml` instead?

Reply:
ok. in that case - i'll move all lambda to aws-lambda instead.